CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+some other effect would make it higher. A player may not receive more than one 
+dose of Dex per engagement. Only works on biologic allies.
+
+==Combat Stim ==
+Adrenaline Reward:	25 Points
+Contagion Reward: 1 Contagion
+Needle Cost: 1 Needle
+Range:			Touch
+Action:			1 Action
+    For 5 turns, the patient moves 2m more per action and gets a -1 to miss with 
+guns. As a side effect, the target suffers a +2 to miss with melee weapons. 
+Increasing the dosage does not increase the effects. This trick only works on 
+biologic patients. 
+
+==Oni ==
+Adrenaline Reward:	25 Points
+Contagion Reward: 1 Contagion
+Needle Cost: 1 Needle
+Range:			Touch
+Action:			1 Action
+    Phentermine derivative manufactured by Sakai Corp on Kimon in sector 5. 
+Central nervous system stimulant. Patient gets a +20 to reflex rolls and a +20 
+to initiative (turn order) for 5 turns. Oni doesn't give more than one turn per 
+round. May only be used on biologic patients. Cannot be used on a patient more 
+than once lest they get mortally constipated, impotent, or their heart explodes. 
+(note: these are the actual effects of a Phentermine overdose) 
+
+==Illegal Stim ==
+Adrenaline Reward:	40 Points
+Contagion Reward: 1 Contagion
+Needle Cost: 1 Needle
+Range: Touch
+Action: 1 Action
+    For 5 turns, the patient moves 3m more per action (max 13), gets a -2 to 
+miss with guns and a +30 to rolls to parkour around obstacles. As a side effect, 
+the target suffers a +2 to miss with melee weapons and a -15 to charisma rolls 
+from being jittery. Increasing the dosage does not increase the effects. This 
+trick only works on biologic patients.  
+
+==Sense Heartbeat ==
+Adrenaline Reward:	20*n Points
+Range:			20m radius centered on the Doc
+Action:			1 Action
+    Reveal active central nervous systems within a 20m radius of the Doc. Every 
+target revealed by Sense Heartbeat that is communicated to an ally and is then 
+defeated grants the Doc 20 adrenaline points. This ability may be sustained for 
+5 adrenaline per turn as a free action.
+
+==Mild Steroid ==
+Adrenaline Reward:	15 Points
+Contagion Reward: 1 Contagion
+Needle Cost: 1 Needle
+Range: Touch
+Action: 1 Action
+    The patient deals an extra 2*(patient's strength) damage with melee weapons, 
+has -1 to miss with melee weapons, and moves +1m per action. As a side effect, 
+the target takes a -1 to both Intelligence and charisma for the duration of the 
+effect. This trick only works on biologic patients. This effect lasts for 4 
+turns.
+
+==Preventative Medicine ==
+Adrenaline Reward:	20 Points
+Range:			Touch
+Action:			1 Action
+Effect:			Target gets a 30 point capacitive shield for the duration of 
+the engagement. This trick works on any race of ally. 
+
+==Nanite Restore ==
+Adrenaline Reward:	n/4
+Range:			Touch
+Action:			Free Action
+Effect:			Transfer n number of Adrenaline points from the Doc to the 
+patient. If This trick is used on another doctor, no adrenaline points are 
+earned. 
```

Comment:
Still have "Nanite Restore" here, restoring adrenaline points from doc to x patient.

If the intention is to give the target X Nanites for X adrenaline points (1 for 1) then I'd need some clarification in the description.

Reply:
I need to replace this with something, I'm just not sure what to replace it with.